nucypher/cli/characters/felix.py

```diff
@@ -14,6 +15,7 @@
 @click.command()
 @click.argument('action')
 @click.option('--teacher-uri', help="An Ursula URI to start learning from (seednode)", type=click.STRING)
+@click.option('--enode', help="An ethereum bootnode enode address to start learning from", type=click.STRING)
```

Comment:
any reason we don't want to call this `--ethnode`?

Reply:
Yeah - perhaps we make this `bootnode`?

Reply:
👍